### PR TITLE
Update umbrella README, add rudimentary Windows section

### PR DESCRIPTION
**Pull request by drandarov-io**
## Description

Update umbrella README.md for readability and structure. Add rudimentary differentiation between Linux, Windows and Mac section.

> As per discussion with @stephanbcbauer 

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by drandarov-io** [comment]
There seems to be an issue with the recent version bump to 0.11.4

> ✖︎ umbrella => (version: "0.11.4", path: "charts/umbrella") > chart version not ok. Needs a version bump!

**Comment by drandarov-io** [comment]
### As noted in the Matrix chat:
When running Minikube through Docker for Desktop on Windows, there is an issue with forwarding any requests through the minikube ip.
Alternatively I tried the native Kubernetes Cluster that is included in Docker for Desktop on Windows. Anyone trying to run Minikube on Windows through Docker for Desktop would have access to it as natively as well.
The steps described in the README also natively work with that Cluster. The only addition is installing the nginx-controller:

```
helm upgrade --install ingress-nginx ingress-nginx --repo https://kubernetes.github.io/ingress-nginx --namespace ingress-nginx --create-namespace

helm install cert-manager jetstack/cert-manager --namespace umbrella --create-namespace --version v1.14.4 --set installCRDs=true

cd charts/umbrella

kubectl apply -f cluster-issuer.yaml

helm install -f your-values.yaml umbrella . --namespace umbrella
```

The only other change is that Docker for Desktops Kubernetes Cluster uses 127.0.0.1 instead of an independent IP. With this configuration I was able to run the umbrella charts unter Windows.

I'm not sure if using the native Docker for Desktop Cluster an option for you but I thought I'd document it at least.

image

image

File: charts/umbrella/cluster-issuer.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+apiVersion: cert-manager.io/v1
```
Comment (evegufy) [inline charts/umbrella/cluster-issuer.yaml:1]: Could you please add a file header?

**Review by evegufy — CHANGES_REQUESTED** [review]
Thanks for the docs PR!
Could you please update (rebase) it to the current state of main and integrate the way you figured out to setup von Windows?

File: charts/umbrella/cluster-issuer.yaml (hunk 1 above)
Comment (drandarov-io) [reply charts/umbrella/cluster-issuer.yaml:1]: Will do

File: charts/umbrella/README.md [hunk 2]
```diff
@@ -35,24 +36,45 @@ Assuming you have a running cluster and your `kubectl` context is set to that cl
 >
 > We plan to test the chart's reliability also on Windows and to update the installation guide accordingly.
```
Comment (evegufy) [inline charts/umbrella/README.md:37]: Maybe change here: `We are working on testing the chart's reliability...`

File: charts/umbrella/README.md [hunk 3]
```diff
@@ -35,24 +36,45 @@ Assuming you have a running cluster and your `kubectl` context is set to that cl
...
 
+### Cluster setup
 
 > Recommendations for resources
 > | CPU(cores) | Memory(GB) |
 > | :--------: | :--------: |
 > |     4      |      6     |
 
+#### Linux
```
Comment (evegufy) [inline charts/umbrella/README.md:46]: Change here to Linux and Mac, as this is also valid for Mac

File: charts/umbrella/README.md [hunk 4]
```diff
@@ -64,8 +86,18 @@ Make sure that the **DNS** resolution for the hosts is in place:
...
 
+To find out the IP address of your Minikube execute:
+
+```bash
+minikube ip
+```
+
+In the following steps, replace `192.168.49.2` with your `minikube ip` if it differs.
+
+#### Linux
```
Comment (evegufy) [inline charts/umbrella/README.md:100]: Change here to Linux and Mac, as this is also valid for Mac

File: charts/umbrella/README.md [hunk 5]
```diff
@@ -93,9 +131,8 @@ If you still face DNS issues afterwards, add the hosts to your /etc/hosts file:
 192.168.49.2    semantics.tx.test
 ```
 
-Replace 192.168.49.2 with your minikube ip.
-
-**Additional network setup** (for Mac only)
+#### Mac
+**Additional network setup**
```
Comment (evegufy) [inline charts/umbrella/README.md:135]: Please call this section Additional network setup for Mac and move it above the Windows section into the Linux and Mac section

File: charts/umbrella/README.md [hunk 6]
```diff
@@ -74,15 +106,21 @@ nameserver 192.168.49.2
 search_order 1
 timeout 5
 ```
-Replace 192.168.49.2 with your minikube ip.
 
-To find out the IP address of your Minikube:
+If you still face DNS issues afterward, add the hosts to your /etc/hosts file:
```
Comment (evegufy) [inline charts/umbrella/README.md:110]: Please remove the word afterward, it's not needed or change to afterwards

File: charts/umbrella/README.md [hunk 7]
```diff
@@ -204,17 +267,20 @@ helm install \
...
 
 ```bash
 helm install \
   --set centralidp.enabled=true,managed-identity-wallet.enabled=true,dataconsumerOne.enabled=true,tx-data-provider.enabled=true \
   umbrella tractusx-dev/umbrella \
   --namespace umbrella
 ```
-Optional:
+
+*Optional:*
```
Comment (evegufy) [inline charts/umbrella/README.md:283]: remove colon

File: charts/umbrella/README.md [hunk 8]
```diff
@@ -255,25 +322,38 @@ Download the chart dependencies:
...
 
-**Data Exchange**
+**Data Exchange Subset**
 
 ```bash
 helm install -f values-adopter-data-exchange.yaml umbrella . --namespace umbrella
 ```
 
-Optional:
+*Optional:*
```
Comment (evegufy) [inline charts/umbrella/README.md:356]: remove colon

**Review by evegufy — CHANGES_REQUESTED** [review]

**Review by evegufy — APPROVED** [review]

**Pull request merged**